app/controllers/home_controller.rb

```diff
@@ -0,0 +1,6 @@
+class HomeController < ApplicationController
+  def index
+    @keywords = Keyword.select_21_link_in_random_order
```

Comment:
I don't like the name of method. we don't have a stringent conditions to select only 21 keyword. This number could be changed in time. So the name of the method should be rename to something like this: 
```
Keyword.in_random_order
```

also @keywords instant variable is not reflecting the value of it. Maybe @random_keyword would be better